animations/videoClip.py

```diff
@@ -116,3 +116,44 @@ def add_text(self, text, font_size, color, font, interline, pos, duration):
         text = TextClip(text, fontsize=font_size, color=color,
                         font=font, interline=interline).set_pos(pos).set_duration(duration)
         self.clip = CompositeVideoClip([self.clip, text])
+
+    def addAudioFromFile(self, audio, start_time, end_time):
```

Comment:
Seems Good. If Pull Request 5 gets accepted, though, this code should be moved as is to BaseClip, so it can work with all clips inheriting from BaseClip.

Reply:
Yea I also thought about that, but to not have to wait, I uploaded it anyways, since the changes aren't much. So when PR 5 gets accepted, I'll move it accordingly.

Reply:
Either way whichever comes first the change will be done. Nice pointer though @TheParodicts